app/src/main/java/com/futurice/freesound/utils/TextUtils.java

```diff
@@ -40,7 +40,7 @@ public static boolean isNullOrEmpty(@Nullable final CharSequence chars) {
      *
      * @return true if the parameter neither null nor empty, false otherwise.
      */
-    public static boolean isNotNullOrEmpty(@Nullable final CharSequence chars) {
+    public static boolean isNotEmpty(@Nullable final CharSequence chars) {
```

Comment:
Create a separate method for `isNotEmpty`, because this implementation also tests for nullability.